### ISSUE TITLE
Can't find stylesheet to import

### ISSUE THREAD

**Issue by Ponkhy** [Spiderpig86/Cirrus#100]
Hey, I wanted to migrate to Cirrus 0.7.0, unfortunately i'm facing this issue rn, maybe someone has a idea what i could do?

![Issue](https://i.imgur.com/5Xns7DY.png)

I'm using Nuxt.js 2.15.8
With Cirrus 0.6.3 i was manually importing every .scss file, but i thougth i should do it now in the good way.

So i tried to import it with
```
@use "node_modules/cirrus-ui/src/cirrus-ext" as * with (
    $config: (
        excludes: (
            ABSOLUTES,
        ),
        opacity: null, // Disable default opacity classes
        extend: (
            // Add your own
            opacity: (
                25: .25,
                50: .5,
                75: .75,
            )
        )
    ),
);
```
and the file is definitely in the node_modules folder :/

**Comment by Spiderpig86** [comment]
Thanks for the bug report! I have reproduced it myself and will be publishing a patch version very soon to fix this with version `0.7.0.1`.

**Comment by Spiderpig86** [comment]
This has been fixed in the new version. Please feel free to install it again.

**Comment by Ponkhy** [comment]
Yes it does work! Thank you very much for the quick fix!

**Comment by art-solopov** [comment]
I'm having a very similar issue on cirrus-ui 0.7.2-patch.1, using SASS CLI 1.75.0:

```
> sass assets/css/app.scss public/app.css

Error: Can't find stylesheet to import.
  ╷
5 │ @use 'sass-true/sass/config/throw';
  │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ╵
  node_modules/cirrus-ui/src/internal/_functions.scss 5:1  @forward
  node_modules/cirrus-ui/src/internal/_index.scss 3:1      @forward
  node_modules/cirrus-ui/src/cirrus-ext.scss 1:1           @use
  assets/css/app.scss 1:1                                  root stylesheet
```

My app.scss:

```scss
@use "../../node_modules/cirrus-ui/src/cirrus-ext" as * with (
  $config: ()
);
```

**Comment by karneaud** [comment]
> I'm having a very similar issue on cirrus-ui 0.7.2-patch.1, using SASS CLI 1.75.0:
> 
> ```
> > sass assets/css/app.scss public/app.css
> 
> Error: Can't find stylesheet to import.
>   ╷
> 5 │ @use 'sass-true/sass/config/throw';
>   │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   ╵
>   node_modules/cirrus-ui/src/internal/_functions.scss 5:1  @forward
>   node_modules/cirrus-ui/src/internal/_index.scss 3:1      @forward
>   node_modules/cirrus-ui/src/cirrus-ext.scss 1:1           @use
>   assets/css/app.scss 1:1                                  root stylesheet
> ```
> 
> My app.scss:
> 
> ```css-scss
> @use "../../node_modules/cirrus-ui/src/cirrus-ext" as * with (
>   $config: ()
> );
> ```

@art-solopov  For me I had to make sure `node_modules` was in the "loadPaths" but I then ran into a similar issue with `@forward ./utils/absolute`

I now have to scrap using this as I just can't get it to work with **SASS >=1.77.***

**Comment by Spiderpig86** [comment]
Is `sass-true` already installed for you?

**Comment by karneaud** [comment]
@Spiderpig86  manually installed it and had to change the syntax 

```
@forward "./absolute";
```

to 

```
@forward "absolute";
```

to get things working again. Just to note that I receive these **DEPRECATION WARNINGS**

> DEPRECATION WARNING: Sass's behavior for declarations that appear after nested

> rules will be changing to match the behavior specified by CSS in an upcoming
> 
> version. To keep the existing behavior, move the declaration above the nested
> 
> rule. To opt into the new behavior, wrap the declaration in `& {}`.
> 
> More info: https://sass-lang.com/d/mixed-decls
> 
>    ┌──> node_modules/cirrus-ui/src/components/forms.scss
> 
> 30 │               background-color: rgba(0, 224, 0, 0.05) !important;
> 
>    │               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ declaration
> 
>    ╵
> 
>    ┌──> node_modules/cirrus-ui/src/internal/_mixins.scss
> 
> 26 │ ┌     &:focus {
> 
> 27 │ │         box-shadow: 0 0 0 0.2rem transparentize(api.fill('success'), 1 - theme.$focus-opacity),
> 
> 28 │ │             inset 0px 1px 8px rgba(0, 0, 0, 0.07);
> 
> 29 │ │     }
> 
>    │ └─── nested rule
> 
>    ╵
> 
>     node_modules/cirrus-ui/src/components/forms.scss 30:13  cirrus-input()
> 
>     node_modules/cirrus-ui/src/components/forms.scss 147:9  @use
> 
>     node_modules/cirrus-ui/src/builds/core.scss 12:1        @forward
> 
>     cirrus-ui/src/cirrus-ext.scss 5:1                       @use

**Comment by Spiderpig86** [comment]
Thanks for letting me know. Let me cut a new issue for the deprecation warning, forwarding paths, etc. It may end up being addressed in 0.8.0 (or even 0.7.2-patch2 if needed) since 0.8.0 is a sizable departure from 0.7.x and before.
